packages/api/cms-api/src/mailer/mailer.module.ts

```diff
@@ -13,10 +13,17 @@ export type MailerModuleConfig = (SMTPTransport | SMTPTransport.Options) & {
 @Global()
 @Module({})
 export class MailerModule {
-    static register(config: MailerModuleConfig): DynamicModule {
+    static async register(config: MailerModuleConfig): Promise<DynamicModule> {
+        const nodemailer = await import("nodemailer");
```

Comment:
is this a good idea, I think the correct/better solution for this would be to extract this module and its dependency into an separate package?
I know you dont want to manage many packages but I think this was a problem long time ago as we didnt have this mono repo release setup like it is now.

For me it would be more logical then if i use module xyz then I've to install optional dep xyz. When we have more such modules this will get really confusing.

Just my thinking about this